src/Config/Type.hs

```diff
@@ -85,6 +87,8 @@ data Classify = Classify
     }
     deriving Show
 
+
+
 -- | A @LHS ==> RHS@ style hint rule.
 data HintRule {- PUBLIC -} = HintRule
```

Comment:
This data type doesn't seem to be public anymore so please drop this comment

Reply:
Ok.